### ISSUE TITLE
How to convert a list into a tuple?

### ISSUE THREAD

**Issue by AshishNamdev** [AshishNamdev/iqa#18]
How to convert a list into a tuple?

- By using Python `tuple()` function we can convert a list into a tuple. But we can’t change the list after turning it into tuple, because it becomers immutable.

### Example:
```python
weekdays = [‘sun’,’mon’,’tue’,’wed’,’thu’,’fri’,’sat’]
listAsTuple = tuple(weekdays)
print(listAsTuple)
```
### Output:
```bash
(‘sun’, ‘mon’, ‘tue’, ‘wed’, ‘thu’, ‘fri’, ‘sat’)
```

**Comment by AshishNamdev** [comment]
Included in [Q10](../blob/master/questions/Q1-50.md#Q10)

**Comment by AshishNamdev** [comment]
included in commit 0ee53c12045d5ea4788e9b4527a052dd89c4cfdd
